src/test-helpers/e2e-helpers/mobile/tapReturn.ts

```diff
@@ -0,0 +1,12 @@
+import { Browser } from 'webdriverio'
+
+/** Tap 'return' on the keyboard. */
+const tapReturn = async (browser: Browser<'async'>) => {
```

Comment:
Let's call this `tapReturnKey` which is a bit less ambiguous. It took me a second to realize what it was referring to.